components/Pamphlet/Pamphlet.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Fragment } from 'react';
+import {
+  Button,
+  Container,
+  Header,
+  Image,
+  List
+} from 'semantic-ui-react';
+
+import styles from './Pamphlet.less';
+
+export const Pamphlet = (props) => {
+  const { title, priceText, description } = props;
+
+  return (
+    <Fragment>
+      <Container className={styles.container}>
+        <Image
+          className={styles.bannerImage}
+          src={props.imageUrl}
+        />
+
+        <div className={styles.containerBody}>
+          <Header
+            as='h3'
+            className={styles.title}>
+            {title}
+          </Header>
+          <p className={styles.priceDifference}>{priceText}</p>
+          <p className={styles.description}>{description}</p>
+
+          <div className={styles.listContainer}>
+            <List>
+              <p className={styles.listDescription}>{'⭐️ You will get:'}</p>
```

Comment:
let's figure out how this props might be handled, I doubt it's going to read `'⭐️ You will get:` for every pamphlet.

Reply:
agreed, this text should be provided by the importing service. It can be a new prop if needed